DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/WallConversionToSpeckle.cs

```diff
@@ -108,21 +104,21 @@ private void AssignHostedElements(Wall target, RevitWall speckleWall)
     }
   }
 
-  private void AssignVoids(Wall target, RevitWall speckleWall)
+  private void AssignVoids(DB.Wall target, SOBR.RevitWall speckleWall)
   {
-    List<CurveArray> voids = GetWallVoids(target);
-    List<Polycurve> polycurves = voids.Select(v => _curveArrayConverter.RawConvert(v)).ToList();
+    List<DB.CurveArray> voids = GetWallVoids(target);
+    List<SOG.Polycurve> polycurves = voids.Select(v => _curveArrayConverter.RawConvert(v)).ToList();
 
     if (polycurves.Count > 0)
     {
       speckleWall["voids"] = polycurves;
     }
   }
 
-  private List<CurveArray> GetWallVoids(Wall wall)
+  private List<DB.CurveArray> GetWallVoids(DB.Wall wall)
   {
-    List<CurveArray> curveArrays = new();
-    var profile = ((Sketch)_contextStack.Current.Document.Document.GetElement(wall.SketchId))?.Profile;
+    List<DB.CurveArray> curveArrays = new();
+    var profile = ((DB.Sketch)_contextStack.Current.Document.Document.GetElement(wall.SketchId))?.Profile;
```

Comment:
Yeah must change this doc type in the context, but TBF I am thinking of taking it out of the context...

Reply:
Comment for this already exists in the contextSTack class.